### PR TITLE
Update

### PR DESCRIPTION
**Pull request by flosse**
I'd recommend to merge all the PRs with "rebase & merge" so that we'll get a clean commit history.

### CODE REVIEW

File: src/bin/simple_test/main.rs [hunk 1]
```diff
@@ -101,8 +101,8 @@ fn simple_test(iface_name: &str) -> i32 {
         Ok(_) => (),
     };
 
-    let try = 40;
-    for _ in 0..try {
+    let r#try = 40;
```
Comment (matwey) [inline src/bin/simple_test/main.rs]: Why should we use `r#try` name using hash sign? As far as I know, `retry` is not a keyword in Rust.

**Comment by matwey** [comment]
@flosse ?

File: src/bin/simple_test/main.rs (hunk 1 above)
Comment (flosse) [reply src/bin/simple_test/main.rs]: This change was made automatically by `cargo fix --edition`.
Now I removed the variable entirely :)

**Comment by matwey** [comment]
Looks good to me. Thank you.

**Pull request merged**